zarr/tests/test_storage.py

```diff
@@ -900,6 +900,16 @@ def test_context_manager(self):
 except ImportError:  # pragma: no cover
     sqlite3 = None
 
+try:
+    import pymongo
+except ImportError:  # pragma: no cover
+    pymongo = None
```

Comment:
What about adding an additional check here for the presence of a local server against which to run tests? E.g.:

```python
try:
    import pymongo
    # check for presence of local server to run tests against
    pymongo.MongoClient(host='127.0.0.1', database='zarr_tests', collection='zarr_tests')
except ImportError:  # pragma: no cover
    pymongo = None
except WhateverExceptionMongoRaisesOnFailedConnection:
    pymongo = None
```

This would ensure tests are skipped if server is not available.

Reply:
done.